### ISSUE TITLE
Clickables: Buttons cannot be empty

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#74]
**Requirement:**
Buttons should include text explaining their functionality, and if icons are used as buttons, a screen-reader only text or an "aria-label" attribute should be used for that description. 

**1 Failed Elements**
```html
<button type="submit" class="button is-ghost" > <svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 24 24" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg"> <path fill="none" d="M0 0h24v24H0z">
```
